tailnet/conn.go

```diff
@@ -347,7 +349,8 @@ var (
 	// process of migrating to. It allows Coder to run alongside Tailscale without conflicts even
 	// if both are set up as TUN interfaces into the OS (e.g. CoderVPN).
 	// fd60:627a:a42b::/48
-	CoderServicePrefix ServicePrefix = [6]byte{0xfd, 0x60, 0x62, 0x7a, 0xa4, 0x2b}
+	CoderServicePrefix     ServicePrefix = [6]byte{0xfd, 0x60, 0x62, 0x7a, 0xa4, 0x2b}
+	CoderServicePrefixCIDR               = netip.MustParsePrefix("fd60:627a:a42b::/48")
```

Comment:
Could do this lazily, ala Tailscale `oncePrefix`

Reply:
How about adding a `AsNetip()` method to the `ServicePrefix` struct instead?

Reply:
I had that, but then I realised we'd be allocating a new one each time we `reconfig`

Reply:
Insignificant compared to how much work a `reconfig` is